CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateCountdown.cpp

```diff
@@ -51,6 +51,7 @@ void Kartaclysm::StateCountdown::Update(const float p_fDelta)
 			pCountdownEvent->SetIntParameter("Disable", 0);
 			for (int i = 0; i < m_iPlayerCount; i++)
 			{
+				// FIXME - this causes a crash when running in debug in VS 2015
 				pHudEvent->SetFloatParameter("Player" + std::to_string(i), (m_vGainsBoost[i] ? 1.3f : 0.0f));
```

Comment:
Project -> CS483 Properties -> Configuration Properties -> General -> Platform Toolset.  Change to an older version.  Though we should fix this at some point, as it means our code doesn't conform to the C++ standard, which is obviously not ideal.